HTML/syntax_test_html.html

```diff
@@ -348,6 +348,14 @@
 ##          ^^ invalid.illegal.bad-comments-or-CDATA
 ##                      ^^^^^^^^^^^^^^^ - invalid.illegal.bad-comments-or-CDATA
 
+        <!-- 1st comment -- >
+##                       ^^^^ - punctuation.definition.comment.end.html
```

Comment:
It may be a good idea to keep negative test selectors a bit more generic in general to reduce the risk of those not applying after scope changes.

Basically just `- punctuation` should be enough here. So unless we change top-level scope the test keeps valid after all kinds of scope name changes.

It won't probably happen here though, but I left too many obsolete test cases behind during larger refactorings in the past to not apply this guideline whenever possible.

Positive tests should be as precise as possible, to find any renamings.